docs/source/development.rst

```diff
@@ -106,14 +106,164 @@ are making:
 
     $ git remote add upstream https://github.com/ga4gh/server.git
 
+All development is done against the ``develop`` branch.  The ``master``
+branch is meant to be kept stable since it is the branch releases are
+based on -- don't touch it!  These are the two mainline branches.
+Our branching model is loosely based on the one described
+`<here http://nvie.com/posts/a-successful-git-branching-model/>`_.
+
+All development should be done in a topic branch.  That is, a branch
+that the developer creates him or herself.  These steps will create
+a topic branch (assuming that ``upstream`` points to 
+``https://github.com/ga4gh/server.git``) and ``origin`` points to 
+``https://github.com/YOUR_GITHUB_USERNAME/server.git``, which can be
+verified by running ``git remote -v``):
 
-**TODO**
+.. code-block:: bash
+
+    $ git fetch --all
+    $ git checkout develop
```

Comment:
I usually do this as
```bash
$ git fetch upstream
$ git checkout upstream/develop
$ git checkout -b TOPIC_BRANCH_NAME
```
What are the advantages/disadvantages of the various approaches? I know there's a million different ways to do things in git, but I think it's worthwhile trying to figure out the most foolproof recipe we can to try and make the learning curve a little less steep.